src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java

```diff
@@ -151,77 +159,131 @@ public void postWithWeirdCharactersIsDisplayedOnDesktopPostDetailsPage() {
-   * |        x |       |             |
-   * |        x |     x |             |
+   * | Category | Title | Description | Open Graph | Content Image |
+   * |          |       |             |            |               |
+   * |          |     x |             |            |               |
+   * |          |       |           x |            |               |
+   * |          |       |             |          x |               |
+   * |          |       |             |            |             x |
+   * |          |     x |           x |            |               |
+   * |          |     x |             |          x |               |
+   * |          |     x |             |            |             x |
+   * |        x |       |             |            |               |
+   * |        x |     x |             |            |               |
+   * |        x |       |           x |            |               |
+   * |        x |       |             |          x |               |
+   * |        x |       |             |            |             x |
+   *
+   *
    * <p>
    * * x - means category was selected or text was added
    */
-  private void assertThatPostWithoutSelectedCategoryAndDescriptionCannotBeAdded(PostsCreator postsCreator) {
-    postsCreator.click()
-        .closeGuidelinesMessage();
+  private void assertThatPostWithoutCategoryOrAnyContentCannotBeAdded(BasePostsCreator postsCreator) {
+    postsCreator.click().closeGuidelinesMessage();
     Assertion.assertFalse(postsCreator.isPostButtonActive());
 
     postsCreator.addTitleWith(TextGenerator.defaultText());
-    Assertion.assertFalse(postsCreator.isPostButtonActive(),
-        "User should not be able to add post with only title filled.");
+    Assertion.assertFalse(
+        postsCreator.isPostButtonActive(),
+        "User should not be able to add post with only title."
+    );
+    postsCreator.clearTitle();
 
     postsCreator.addDescriptionWith(TextGenerator.defaultText());
-    Assertion.assertFalse(postsCreator.isPostButtonActive(),
-        "User should not be able to add post with title and description filled.");
+    Assertion.assertFalse(
+        postsCreator.isPostButtonActive(),
+        "User should not be able to add post with only description."
+    );
+    postsCreator.clearDescription();
+
+    postsCreator.addDescriptionWithLink("http://fandom.wikia.com");
+    Assertion.assertFalse(
+        postsCreator.isPostButtonActive(),
+        "User should not be able to add post with only open graph."
+    );
+    postsCreator.clearDescription();
+    postsCreator.clearOpenGraph();
+
+    postsCreator.uploadImage();
+    Assertion.assertFalse(
+        postsCreator.isPostButtonActive(),
+        "User should not be able to add post with only content image uploaded."
+    );
+    postsCreator.removeImage();
 
+    postsCreator.addTitleWith(TextGenerator.defaultText());
+    postsCreator.addDescriptionWith(TextGenerator.defaultText());
+    Assertion.assertFalse(
```

Comment:
Should be?

Reply:
Nah, this test checks negative cases - when user **can't** add the post. In this case, it's a post with title and description but without category.